pcsx2/VMManager.cpp

```diff
@@ -2981,7 +2982,13 @@ void VMManager::WarnAboutUnsafeSettings()
 	if (EmuConfig.GS.AccurateBlendingUnit <= AccBlendLevel::Minimum)
 	{
 		append(ICON_FA_BLENDER,
-			TRANSLATE_SV("VMManager", "Blending is below basic, this may break effects in some games."));
+			TRANSLATE_SV("VMManager", "Current Blending Accuracy is below Basic, this may break effects in some games."));
```

Comment:
Maybe `Blending Accuracy is below Basic, this may break effects in some games.` instead?